railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -58,3 +58,11 @@
 
 # Raise when running tests if fixtures contained foreign key violations
 # Rails.application.config.active_record.verify_foreign_keys_for_fixtures = true
+
+# Change the variant processor for Active Storage.
+# Changing this default means updating all places in your code that
+# generate variants to use image processing macros and ruby-vips
+# operations. A full migration guide is available:
+# LINK_TO_MIGRATION_GUIDE_HERE
```

Comment:
you could roll your upgrade guide PR into this one?

Reply:
Since upgrading could be a reasonably large job, I think it's worth calling out that it's also fine to keep the current version.

Reply:
In the end I did not open a PR for the migration guide. It evolved into this one. After our conversation I decided that if I felt so strongly about `:vips` I should go all the way and suggest changing the default.

I've simplified the text here, and added an entry in the "Upgrading Ruby on Rails" guide. However, I'm still unsure where I should put the upgrade guide. Any suggestions?

Reply:
Could it fit in the upgrading guide, or would it be way too long?

Reply:
I've added it to the upgrading guide. Not sure how to judge if it's too long or not though. Is there a way to view how this will look like when it's published?